internal/sharding/sharding.go

```diff
@@ -22,6 +22,12 @@ func (c Config) RelabelConfigs() []promcfg.RelabelConfig {
 	return []promcfg.RelabelConfig{
 		{
 			SourceLabels: []string{"__address__"},
+			Regex:        `(\d{1,3}\.\d{1,3}\.\d{1,3}.\d{1,3})(?::\d+)?`,
```

Comment:
I think this is a good solution, all targets from a Pod will be evaluated in the same shard.

nit: IIRC we have documented which was the hashed value, so it might be worth to update that in the docs.

Reply:
Perhaps a regex like the one is [being used](https://github.com/newrelic/newrelic-prometheus-configurator/pull/228/files#diff-2d5fd57ede2c4058278f85eb9e1a145957d8123296cdc4c20790da352a195703R58) at the port replacement face could make this easier to read and cover ipv6?

Reply:
I updated the docs as below
- https://newrelic.atlassian.net/wiki/spaces/INST/pages/2561344375/Prometheus+configurator+in+k8s+II+MVP
- https://newrelic.atlassian.net/wiki/spaces/INST/pages/2427125821/CDD+Prometheus+Configurator+in+K8s

Reply:
> I think this is a good solution, all targets from a Pod will be evaluated in the same shard.
> 
> nit: IIRC we have documented which was the hashed value, so it might be worth to update that in the docs.

I updated the docs as below

https://newrelic.atlassian.net/wiki/spaces/INST/pages/2561344375/Prometheus+configurator+in+k8s+II+MVP
https://newrelic.atlassian.net/wiki/spaces/INST/pages/2427125821/CDD+Prometheus+Configurator+in+K8s

Reply:
something like this might work https://regex101.com/r/McCxYm/1.
I was referring to this external docs https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/scale-prometheus-instances/#horizontal